javaScript/Running.js

```diff
@@ -0,0 +1,31 @@
+// let Navlinking = document.querySelector('.newNavLinks');
+// const Offing = document.getElementById('newCloseTag');
```

Comment:
[OPTIONAL] Please remove the unwanted comments from your codebase because they make your codebase messy if you want to check your previous code then check your GitHub commit history :slightly_smiling_face: